components/web/apps/org.wso2.carbon.apimgt.publisher/src/main/fragments/global-endpoints-listing/global-endpoints-listing.hbs

```diff
@@ -0,0 +1,32 @@
+{{js "js/index.js"}}
+    <div class="row">
+            <div id="endpoint-listing-welcome-message">
+                <div class="message message-success">
+                    <h4><i class="icon fw fw-success"></i>No Endpoints to list</h4>
+                    <p>You don't have any Endpoints to list here.Click on the <b>ADD</b> button above to add new Endpoint.</p>
```

Comment:
Add localization support to "p>You don't have any Endpoints to list here.Click on the <b>ADD</b> button above to add new Endpoint.</p>"